src/Dapr.Actors/Dapr.Actors.csproj

```diff
@@ -1,26 +1,50 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
-  </PropertyGroup>
+    <PropertyGroup>
```

Comment:
@jigargandhi - what are the changes in this file accomplishing? I'm not sure how this is related to adding the new API.

Reply:
Most of the changes in file are formatting changes because I opened it in vscode. 
I have added InternalsVisibleToAttribute to ease mocking because some types are internal and